src/UI/Components/TreeTable/TreeRow/InlineInput.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Button, Icon, Switch, TextInput } from "@patternfly/react-core";
+import { CheckIcon } from "@patternfly/react-icons";
+import styled from "styled-components";
+import { words } from "@/UI/words";
+const InlineInput = ({
```

Comment:
add new line between imports and start of your code

Reply:
added